LDAR_Sim/model_code/methods/deployment/mobile_company.py

```diff
@@ -0,0 +1,132 @@
+import pandas as pd
```

Comment:
Is "mobile_company" the right name for this file? 

I wonder if we should put everything related to scheduling in the same folder or same file called scheduling

Reply:
same as above